test/bin.js

```diff
@@ -37,7 +37,7 @@ t.test('finds node', function (t) {
     t.equal(signal, null)
     t.equal(code, 0)
     t.equal(err, '')
-    t.match(out, /[\\\/]node(\.exe)?$/)
+    t.match(out, /[\\\/]node(\.(exe|EXE))?$/)
```

Comment:
Probably better to just add `/i` to these regexps, since it could just as easily be `NODE.EXE`.